src/infrastructure/validation/logger/LogLevel.java

```diff
@@ -1,15 +1,22 @@
-/**
- * 
- */
 package infrastructure.validation.logger;
 
 /**
+ * The ILogger interface depends on the LogLevel enum
+ * The LogLevel enum is used to specify the type of log used:
+ * 
```

Comment:
Similar hanging ':' comment as given in below comment

Reply:
yet another :(
really need to be careful next time doing batch editing